packages/bun-error/index.tsx

```diff
@@ -827,7 +785,6 @@ const NativeStackTrace = ({
           highlightColumnStart={position.column_start}
           buildURL={buildURL}
           highlightColumnEnd={position.column_stop}
-          isClient={isClient}
```

Comment:
There was a reason for this, but I don't remember it anymore

Reply:
`SourceLines` doesn't currently use the prop. Should I put this back, in case there are future plans for it?

Reply:
no this is fine